api/supported.go

```diff
@@ -35,12 +35,8 @@ func GetSupportedClusterList(c *gin.Context) {
 				Key:  constants.Google,
 			},
 			{
-				Name: "Build Your Own Cluster",
-				Key:  constants.BYOC,
-			},
-			{
-				Name: "Dummy cluster",
-				Key:  constants.Dummy,
+				Name: "Kubernetes Cluster",
+				Key:  constants.Kubernetes,
```

Comment:
Are you sure you want to remove the Dummy cluster?

Reply:
yes unless you want to promote it on the UI